src/app/components/home/LoanPanel.tsx

```diff
@@ -25,6 +26,7 @@ const LoanPanel = () => {
 
   const stakedICXAmount = useDepositedValue();
   const loanBorrowedValue = useLoanBorrowedValue();
+  const walletBalance = useWalletBalanceValue();
   const totalLoanAmount = stakedICXAmount.div(4).minus(loanBorrowedValue);
```

Comment:
totalLoanAmount = (stakedICXAmount / 4) - loanBorrowedValue
I think it is not correct formula.
totalLoanAmount = (stakedICXAmount /5)

Reply:
max you can borrow is 1/4 of collateral.
Max you can borrow and receive reward is 1/5 of collateral, so at 1/4 you can still borrow but not get a reward.

To get remaining amount you can borrow, take max you can borrow, and subtract already borrowed amount

Reply:
I thought the collateral ratio was 500%.

Reply:
I think we need to calculate the stakedICXAmount in USD.
stakedICXAmount * (sICX/USD price) / 4 - loanBorrowedValue

Reply:
If the sICX(or ICX) price drops, the borrowed amount is bigger than the total borrowable available amount. In this case, the available amount is minus. In this case, how will handle that UI?
![image](https://user-images.githubusercontent.com/50083174/111968800-83c1ab80-8b34-11eb-80d2-344a4660b8fb.png)

Reply:
if less than 0, you can display 0